src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -15,6 +15,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_TASK = new Prefix("task/");
     public static final Prefix PREFIX_DEADLINE = new Prefix("by/");
     public static final Prefix PREFIX_TO = new Prefix("to/");
+    public static final Prefix PREFIX_FROM = new Prefix("from/");
     public static final Prefix PREFIX_EFFICIENCY = new Prefix("eff/");
     public static final Prefix PREFIX_TASK_OWNER = new Prefix("o/");
```

Comment:
the prefix could be better. Maybe something like by instead of o as o is quite vague